README.md

```diff
@@ -14,22 +14,5 @@ In order to use our tool, you have to install it in your Squeak image. Our prefe
 - For later updates, you can just pull the new version from the Git Repository. To do that, select the `dev` branch and click the button <kbd>Pull</kbd>.
 - Once the new version was downloaded, right click the latest commit and choose <kbd>Checkout objects</kbd>.
 
-### Metacello
-- Open a workspace and execute
-```smalltalk
-Metacello new
-    baseline: 'ChangesReloaded';
-    repository: 'github://hpi-swa-teaching/ChangesBrowser:dev/packages';
-    load
-```
-as well as
-```smalltalk
-Metacello new
-    baseline: 'Changes';
-    repository: 'github://hpi-swa-teaching/ChangesBrowser:dev/packages';
-    load
-```
-- This is necessary because the 2020 project supports the 2017 project, but the browsers themselves have their own baselines. You need both for full functionality.
-
```

Comment:
Is the removal of the Metacello section intentional?

Reply:
Nope, just some happy Squeak merge incident